g2p/tests/public/data/fin.psv

```diff
@@ -0,0 +1,6 @@
+fin|fin-ipa|nähdä|næhdæ
+fin|fin-ipa|polkupyörä|polkupyøræ
+fin|fin-ipa|haluttaa|haluttɑː
+fin|fin-ipa|tulevaisuus|tuleʋɑjsuːs
+fin|fin-ipa|pitämättömyyksien|pitæmættømyːksjen
+fin|fin-ipa|mihinkään|mihiŋkæːn
```

Comment:
I would add some lines to unit test fin -> eng-ipa and/or fin -> eng-arpabet too. I find the generated mappings need a bit of testing too, I've come across bugs when testing the chain.